src/Text/Pandoc/Readers/LaTeX.hs

```diff
@@ -2383,16 +2383,26 @@ parseTableRow envname prefsufs = do
   cells <- mapM (\ts -> setInput ts >> parseTableCell) rawcells
   setInput oldInput
   spaces
-  return $ Row nullAttr cells 
+  -- Because of table normalization performed by Text.Pandoc.Builder.table, 
+  -- we need to remove empty cells
+  return $ Row nullAttr $ filter (\c -> c /= emptyCell) cells
```

Comment:
I'm a little confused by this comment; could you explain what is going on?

Reply:
esp. in light of the comment at 2403

Reply:
The comment above at line 2386-2387 refers to the fact that the `Text.Pandoc.Builder.table` function expects the table representation to be different than LaTeX. Consider a 2x2 table, with a 2x1 cell on the left:

```
+---+---+
| 1 | 2 |
+   +---+
| 1 | 3 |
+---+---+
```

`Text.Pandoc.Builder.table` takes in a list of rows, and the first row has an element which spans two rows. Therefore, since its a 2x2 table, only one element is required in the second row. `Text.Pandoc.Builder.table` will take the first element of the second row, and reject all "extras".

Contrast this to LaTeX, which represents the table above as follows:

```latex
\begin{tabular}{cc}
   \multirow{2}{1em}{1} & 2 \\
                                     & 3 \\
\end{tabular}
```
with an empty cell in the bottom left corner. This is what the comment at 2403 is about (as well as #6603) : syntax of LaTeX tables **requires** the presence of empty cells. Table cells that span more than one row/column must be accompanied by appropriate number of empty cells.

Therefore, to "translate" between what is syntactically-correct LaTeX and what `Text.Pandoc.Builder.table` expects, empty cells must be pruned.

Reply:
I see, thanks